app/lib/ui/colors.dart

```diff
@@ -22,3 +22,10 @@ MaterialColor createMaterialColor(Color color) {
   }
   return MaterialColor(color.value, swatch);
 }
+
+extension CustomColorScheme on ColorScheme {
```

Comment:
This is a neat trick.
And maybe it's just me but do the colors look slightly off to you as well? I feel like it does not really fit the existing color scheme just yet. But then again maybe it's just that I need to get used to more than just 2 colors that are used in the theme right know :smile: